docs-2.0/nebula-studio/deploy-connect/st-ug-connect.md

```diff
@@ -9,28 +9,29 @@ Before connecting to the NebulaGraph database, you need to confirm the following
 - The NebulaGraph services and Studio are started. For more information, see [Deploy Studio](st-ug-deploy.md).
 
 - You have the local IP address and the port used by the Graph service of NebulaGraph. The default port is `9669`.  
-  
-  !!! note
-
-        Run `ifconfig` or `ipconfig` on the machine to get the IP address.
 
 - You have a NebulaGraph account and its password.
 
-  !!! note
-
-        If authentication is enabled in NebulaGraph and different role-based accounts are created, you must use the assigned account to connect to NebulaGraph. If authentication is disabled, you can use the `root` and any password to connect to NebulaGraph. For more information, see [NebulaGraph Database Manual](https://docs.nebula-graph.io/).
-
 ## Procedure
 
 To connect Studio to NebulaGraph, follow these steps:
 
-1. On the **Config Server** page of Studio, configure these fields:
+1. Type `http://<ip_address>:7001` in the address bar of your browser.
+
+  The following login page shows that Studio is successfully connected to NebulaGraph.
+
+  ![NebulaGraph Studio Login page](https://docs-cdn.nebula-graph.com.cn/figures/studio_login_230130_en.png)
 
-  - **Host**: Enter the IP address and the port of the Graph service of NebulaGraph. The valid format is `IP:port`. The default port is `9669`.  
+2. On the **Config Server** page of Studio, configure these fields:
+
+  - **Graphd IP address**: Enter the IP address of the Graph service of NebulaGraph. For example, `192.168.10.100`.
 
     !!! note
 
-        When NebulaGraph and Studio are deployed on the same machine, you must enter the IP address of the machine, but not `127.0.0.1` or `localhost`, in the **Host** field.
+        - When NebulaGraph and Studio are deployed on the same machine, you must enter the IP address of the machine, but not `127.0.0.1` or `localhost`.
```

Comment:
```suggestion
        - When NebulaGraph and Studio are deployed on the same machine, you must enter the IP address of the machine, instead of `127.0.0.1` or `localhost`.
```